packages/react-native-editor/src/test/index.test.js

```diff
@@ -176,9 +176,19 @@ describe( 'Register Gutenberg', () => {
 		expect( hookCallOrder ).toBeGreaterThan( onRenderEditorCallOrder );
 	} );
 
-	it( 'initializes the editor', () => {
-		const { getByTestId } = initGutenberg();
-		const blockList = waitFor( () => getByTestId( 'block-list-wrapper' ) );
+	it( 'initializes the editor', async () => {
+		const MockEditor = () => <Text>Mock Editor</Text>;
+		jest.mock( '../setup', () => {
+			return {
+				__esModule: true,
+				default: jest.fn( () => <MockEditor /> ),
+			};
+		} );
```

Comment:
This test failed when run in isolation, as it appeared dependent upon the mock of `../setup` found in sibling tests.

Reply:
@fluiddot I leveraged a mock of `../setup` as a quick fix to avoid errors from additional dependencies of the `../setup` component. I think asserting against a deeper render makes a lot of sense. 👍🏻

Reply:
> In any case, and in order to prevent blocking the PR and Jest upgrade, we could stick with this change and tackle my original idea in a future PR 👍 .

@dcalhoun heads up that I created [a PR](https://github.com/WordPress/gutenberg/pull/37955) related to this comment.